README.md

```diff
@@ -60,131 +80,50 @@ systemctl start docker
-Alternatively you can run `yast2_container -l` command to list the supported
-modules.
-
-The container image contains some more modules which are not displayed in the
-control center. These modules have been installed by dependencies but they will
-probably not work properly, do not use them.
-
-## Implementation Details
-
-### Accessing the Host System
-
-The host system is mounted to `/mnt` directory in the container. YaST must use
-this subdirectory instead of the root directory for reading/writing files.
-
-But that's usually not supported, esp. for modules designed to run only in
-an installed system.
+If you want to see the command which will be executed in advance then use the
+`--display` option:
 
-### Package Management
-
-Libzypp supports installing into a chroot directory, that's used during standard
-installation. That means the package management should work fine.
-
-But the problem is that YaST needs to distinguish between the packages needed
-in the management container and the packages needed in the host system.
-
-The partitioning tools could be used from the container, but on the other hand,
-configuring NetworkManager requires it in the host system so it can set the
-network after reboot.
-
-Unfortunately this distinction is missing in YaST...
-
-### Executing commands
-
-Similarly to the package management, it is important to know where the executed
-command is available and where it needs to run.
-
-- Run the command in the container (e.g. `fdisk` can be used from a container
-  and does not need access to the host files)
-- Run it in the container and pass it `/mnt` option if that is possible so
-  it modifies the host system
-- Run it in the container but copy the result into the target system
-- Run it in the host system using `chroot` (but that requires the command to be
-  available in the host system)
-- Run the command using SSH, this would ensure it is fully executed in the host
-  system context, but that would require SSH to be installed and running.
-
-### Other Interactions with the System
-
-It is a question which other interactions with the host system are possible
-or not. It turned out that even loading kernel modules works with
-`chroot /mnt modprobe <module>`...
+```shell
+podman container runlabel --display run registry.opensuse.org/suse/alp/workloads/tumbleweed_containerfiles/suse/alp/workloads/yast-mgmt-ncurses:latest
+```
 
-### Logging and Locking
+### Docker
 
-The YaST log is written into `/var/log/YaST2/y2log` in the host as usually.
+Docker does not support using the `RUN` label directly. We need to manually
+extract it, modify it for Docker and then start:
 
-The libzypp lock (`/var/run/zypp.pid`) is also created in the hosts system,
-this avoids running multiple instances of the package management at once.
+```shell
+# the image to use
+IMAGE="registry.opensuse.org/suse/alp/workloads/tumbleweed_containerfiles/suse/alp/workloads/yast-mgmt-ncurses:latest"
```

Comment:
I discuss it with fcrozat and the plan is to have shorter url in future. Just not priority now

Reply:
I see, thanks for the info.